src/vis/vis.js

```diff
@@ -135,16 +135,21 @@ var VisModel = Backbone.Model.extend({
     var windshaftClient = new WindshaftClient(windshaftSettings);
 
     // Create the public Analysis Service
-    this.analysisService = new AnalysisService({
+    this._analysisService = new AnalysisService({
       vis: this,
       apiKey: this.get('apiKey'),
       authToken: this.get('authToken'),
       layersCollection: this._layersCollection,
       dataviewsCollection: this._dataviewsCollection
     });
 
-    // Public namespace exposing public methods.
-    this.analysis = this.analysisService;
+    // Public wrapper exposing public methods.
+    this.analysis = {
```

Comment:
Why not expose the instance of the `AnalysisService` as we're doing before?

Reply:
To expose only the public methods and nothing more in the wrapper/namespace. But it works pointing to the analysisService instance also.